_tour/tuples.md

```diff
@@ -22,7 +22,7 @@ Tuples come in handy when we have to return multiple values from a function.
 A tuple can be created as:
 
 ```tut
-val ingredient = ("Sugar" , 25):Tuple2[String, Int]
+val ingredient : Tuple2[String, Int] = ("Sugar" , 25)
```

Comment:
I guess the old syntax is also valid, it is a [typed expression](https://www.scala-lang.org/files/archive/spec/2.12/06-expressions.html#typed-expressions).